src/serializer/LazyObjectsSerializer.js

```diff
@@ -105,6 +118,121 @@ export class LazyObjectsSerializer extends ResidualHeapSerializer {
     return getOrDefault(this._valueLazyIds, obj, () => this._lazyObjectIdSeed++);
   }
 
+  _isLazyObject(val: Value) {
+    // Currently, only support lazy raw object.
+    return this._isRawObject(val);
+  }
+
+  // Must keep in-sync with ResidualHeapSerializer logic.
```

Comment:
Kept in sync with what exactly?

Reply:
Keep in sync with "_serializeValue()" of ResidualHeapSerializer. Will add.